src/client/src/client/cluster.rs

```diff
@@ -19,7 +19,7 @@ use super::*;
 /// An interface for interacting with Fluvio streaming
 pub struct Fluvio {
     socket: AllMultiplexerSocket,
-    config: ClientConfig,
+    pub config: ClientConfig,
```

Comment:
Why do you need this to be public? If it's really necessary, can we mark it as `#[doc(hidden)]`?

Reply:
Commit 571ab32 removes `pub` visibility. Removed the need for it in the node client.